src/generic/FormSwitchGroup.jsx

```diff
@@ -25,7 +25,7 @@ export default function FormSwitchGroup({
       className={className}
     >
       <div className="d-flex flex-column">
-        <div className="d-flex flex-row justify-content-between align-items-center pb-2 z-index-3">
```

Comment:
@mehaknasir why this is removed?

Reply:
dialog modal upper border shadow padding was still affecting the switch button. that's why I added `z-index` for switch button.

Reply:
oh yes I will check if that issue is fixed with my changes then we are good to go with these changes

Reply:
@awaisdar001 this issue is fixed in paragon. so we dont need this fix here.